src/Tribe/Views/V2/Views/By_Day_View.php

```diff
@@ -147,6 +148,73 @@ public function get_grid_days( $date = null, $force = false ) {
 				$repository = tribe_events( 'period' );
 			}
 			$repository->by_period( $grid_start_date, $grid_end_date )->fetch();
+		} else {
+			global $wpdb;
+
+			$first_grid_day = $days->start;
+			$start          = tribe_beginning_of_day( $first_grid_day->format( Dates::DBDATETIMEFORMAT ) );
+			$last_grid_day  = $days->end;
+			$end            = tribe_end_of_day( $last_grid_day->format( Dates::DBDATETIMEFORMAT ) );
+
+			$day_ids = tribe_events()
+				->set_found_rows( true )
+				->fields( 'ids' )
+				->by_args( $repository_args )
+				->where( 'date_overlaps', $start, $end, null, 2 )
+				->per_page( -1 )
+				->order_by( $order_by, $order )
+				->all();
+
+			$day_results = [];
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: d30eeed8796f59785823d1e1b975d9d3</sub>